lib/samples/query_related_features/query_related_features.dart

```diff
@@ -0,0 +1,280 @@
+                const Divider(),
+                // Display the list of feature preserves for the selected park.
+                Flexible(
+                  child: ListView.builder(
+                    shrinkWrap: true,
+                    padding: EdgeInsets.zero,
+                    itemCount: _featurePreserves.length,
+                    itemBuilder: (context, index) {
+                      return ListTile(
+                        dense: true,
+                        contentPadding: EdgeInsets.zero,
+                        title: Text(
+                          _featurePreserves[index],
+                          style: Theme.of(context).textTheme.titleSmall,
+                        ),
+                      );
+                    },
+                  ),
+                ),
+                const Divider(),
+                Text(
+                  'Alaska National Species',
+                  style: Theme.of(context).textTheme.titleMedium,
+                ),
+                const Divider(),
+                // Display the list of feature species for the selected park.
+                Flexible(
+                  child: ListView.builder(
+                    shrinkWrap: true,
+                    padding: EdgeInsets.zero,
+                    itemCount: _featureSpecies.length,
+                    itemBuilder: (context, index) {
+                      return ListTile(
+                        dense: true,
+                        contentPadding: EdgeInsets.zero,
+                        title: Text(
+                          _featureSpecies[index],
+                          style: Theme.of(context).textTheme.titleSmall,
+                        ),
+                      );
+                    },
+                  ),
+                ),
+              ],
+            ),
+          );
+  }
+
+  void onMapViewReady() async {
+    // Create a map with a topographic basemap style.
+    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
+
+    // Feature table for the Alaska National Parks layer
+    _alaskaNationalParksFeaturesTable = ServiceFeatureTable.withUri(
+      Uri.parse(
+        'https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/AlaskaNationalParksPreservesSpecies_List/FeatureServer/1',
+      ),
+    );
+
+    // Create parks feature layer, the origin layer in the relationship
+    _alaskaNationalParksLayer =
+        FeatureLayer.withFeatureTable(_alaskaNationalParksFeaturesTable);
+
+    // Add parks feature layer to the map
+    map.operationalLayers.add(_alaskaNationalParksLayer);
+    await _alaskaNationalParksLayer.load();
+
+    // Feature table for related preserves layer
+    _alaskaNationalParksPreservesTable = ServiceFeatureTable.withUri(
+      Uri.parse(
+        'https://services2.arcgis.com/ZQgQTuoyBrtmoGdP/ArcGIS/rest/services/AlaskaNationalParksPreservesSpecies_List/FeatureServer/0',
+      ),
+    );
+    // Feature table for related species layer
```

Comment:
Add "Create a feature table..." to keep the verb driving these comments consistently.